libkineto/ipcfabric/FabricManager.h

```diff
@@ -91,7 +97,7 @@ struct Message {
   std::string src;
 };
 
-#ifdef ENABLE_IPC_FABRIC
+#if !defined(KINETO_NAMESPACE) || defined(ENABLE_IPC_FABRIC)
```

Comment:
Why is this not defined for Kineto Namespace?

Reply:
I just wanted to make files in dynolog and kineto the same for convenience. So !defined(KINETO_NAMESPACE) is just a check for "compiling inside dynolog". Alternatively I can define ENABLE_IPC_FABRIC in dynolog's build if you prefer that